src/uct/ib/base/ib_verbs.h

```diff
@@ -270,6 +270,12 @@ static inline int ibv_exp_cq_ignore_overrun(struct ibv_cq *cq)
 #if IBV_HW_TM
 #  if HAVE_INFINIBAND_TM_TYPES_H
 #    include <infiniband/tm_types.h>
+struct ibv_ravh {
```

Comment:
why do we define if tm_types.h exists only?

Reply:
if defined IBV_HW_TM and no defined HAVE_INFINIBAND_TM_TYPES_H (MOFED case) - verbs_exp.h provides ibv_exp_tmh_ravh.

Reply:
ic, did not notice folded `define ibv_ravh                        ibv_exp_tmh_ravh`  below